includes/definitions/discovery/moxa-etherdevice.yaml

```diff
@@ -1,4 +1,3 @@
-mib: IF-MIB:MOXA-IKS6726A-MIB:MOXA-EDSG508E-MIB:MOXA-EDSP510A8POE-MIB
```

Comment:
You should keep IF-MIB here cause it is used for the description building : 
   'sensor_descr' => '{{ $ifDescr }} SFP module Transmit Power'
That explains why your SFP module transmit power lost its description.